src/test/java/com/powsybl/openloadflow/sensi/ac/AcSensitivityAnalysisTest.java

```diff
@@ -918,14 +918,15 @@ void testWithHvdcAcEmulation() {
 
         LoadFlowParameters parameters = new LoadFlowParameters();
         parameters.setBalanceType(LoadFlowParameters.BalanceType.PROPORTIONAL_TO_GENERATION_P_MAX);
+        parameters.setHvdcAcEmulation(true);
         OpenLoadFlowParameters.create(parameters)
-                .setSlackBusSelectionMode(SlackBusSelectionMode.MOST_MESHED)
-                .setHvdcAcEmulation(true);
+                .setSlackBusSelectionMode(SlackBusSelectionMode.MOST_MESHED);
 
         SensitivityAnalysisParameters sensiParameters = createParameters(false);
+        sensiParameters.setLoadFlowParameters(parameters);
         List<SensitivityFactor> factors = List.of(createBranchFlowPerInjectionIncrease("l25", "d2"));
 
         SensitivityAnalysisResult result = sensiRunner.run(network, factors, Collections.emptyList(), Collections.emptyList(), sensiParameters);
-        assertEquals(0.499, result.getBranchFlow1SensitivityValue("d2", "l25"), LoadFlowAssert.DELTA_POWER);
+        assertEquals(0.442, result.getBranchFlow1SensitivityValue("d2", "l25"), LoadFlowAssert.DELTA_POWER);
```

Comment:
Why do we have a different behavior?

Reply:
Because my unit test was wrong. See the loadflow parameters: they were not attached. So the default value of HVDC AC emulation was `false` in the OLF parameters...